### PR TITLE
Tweak README labels

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +59,10 @@ As an open source project, Ballerina welcomes contributions from the community.
 
 Check the issue tracker for open issues that interest you. We look forward to receiving your contributions.
 
+## Code of Conduct
+
+All contributors are encouraged to read [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).
+
 ## Useful links
```
Comment (shafreenAnfar) [inline README.md:66]: Should we make it `Useful Links` ?

File: README.md [hunk 2]
```diff
@@ -26,7 +23,7 @@ For a quick sample on demonstrating the usage see [Ballerina By Example](https:/
 
 ## Building from the source
```
Comment (shafreenAnfar) [inline README.md:24]: Should we make change this to `Building from The Source` ?

Comment (niveathika) [reply README.md:24]: Isn't it `Building from the Source`? Should we capitalise the `the` too?

File: README.md (hunk 1 above)
Comment (niveathika) [reply README.md]: Fixed in 0cd2273

File: README.md (hunk 2 above)
Comment (niveathika) [reply README.md]: Fixed it as `Building from the Source` in 0cd2273

**Review by shafreenAnfar — APPROVED** [review]

**Pull request merged**